rfb/rfbclient.h

```diff
@@ -370,13 +370,13 @@ typedef struct _rfbClient {
 	LockWriteToTLSProc LockWriteToTLS;
 	UnlockWriteToTLSProc UnlockWriteToTLS;
 
-        /* Hooks for custom rendering
+        /** Hooks for custom rendering
          *
          * VNC rendering boils down to 3 activities:
          * - GotCopyRect: copy an area of the framebuffer
          * - GotFillRect: fill an area of the framebuffer with a solid color
          * - GotBitmap: copy the bitmap in the buffer into the framebuffer
-	 * The client application should either set all three of these or none!
+         * The client application should either set all three of these or none!
```

Comment:
Can you give me more detailed instructions on how to do that? This is the first time I try to contribute to a project, so I'm not familiar with the procedure.